tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -940,6 +940,21 @@ def test_spline_filter1d_output(self, xp, scp, array_order):
         return output
 
 
+# See #5537
+@testing.slow
+@testing.with_requires('scipy')
+class TestSplineFilter1dLargeArray:
+
+    @pytest.mark.parametrize('mode', ['mirror', 'grid-wrap', 'reflect'])
+    @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5, scipy_name='scp')
+    def test_spline_filter1d_large_array(self, xp, scp, mode):
+        if mode == 'grid-wrap' and scipy_version < '1.6.0':
+            pytest.skip('testing mode grid-wrap requires scipy >= 1.6.0')
+        x = testing.shaped_random(
+            (2**10, 2**10, 2**10), dtype=numpy.float64, xp=xp)
```

Comment:
@takagi Can we reduce the memory usage, e.g. using numpy.float32 or 16? Jenkins is failing by OOM.

Reply:
Sure, I'll fix it!